docs/auth/identity-resolver.md

```diff
@@ -403,6 +407,22 @@ async signInResolver({ profile }, ctx) {
 }
 ```
 
+##### Using the `dangerouslyAllowSignInWithoutUserInCatalog` Option
```

Comment:
Can we include some clear details on the impact of turning this on? Just to help those who might use this better understand why it's prefixed with `dangerouslyAllow` 👍

Reply:
Sounds good and thanks for taking a look! I've added a simple explaination, I'm conflicted on how much (technical) detail to add here, what do you think?